packages/spor-react/src/theme/components/button.ts

```diff
@@ -60,25 +71,61 @@ const config = defineStyleConfig({
       _active: {
         backgroundColor: "azure",
       },
-    },
-    secondary: {
-      backgroundColor: "coralGreen",
-      color: "darkTeal",
+    }),
+    secondary: (props) => ({
+      // FIXME: Update to use global defined background color for darkMode whenever it is available instead of alpha
+      backgroundColor: mode("seaMist", "whiteAlpha.100")(props),
+      color: mode("darkTeal", "white")(props),
+      // order is important here for now while we do not have global defined background color for darkMode
+      _hover: {
+        backgroundColor: mode("coralGreen", "whiteAlpha.200")(props),
+      },
       ...focusVisible({
         focus: {
-          boxShadow: `inset 0 0 0 4px ${colors.coralGreen}, inset 0 0 0 4px ${colors.coralGreen}, inset 0 0 0 6px currentColor`,
+          boxShadow: `inset 0 0 0 2px ${mode(
+            colors.greenHaze,
+            colors.azure,
+          )(props)}, inset 0 0 0 4px ${mode(
+            colors.white,
+            "rgba(0,0,0,0.34)",
```

Comment:
Can this be either `colors.blackAlpha[300]` or `colors.blackAlpha[400]`?

Reply:
It could, but wouldn't look as nice. Visually, its probably negligible as it's very small.

Reply:
Used `blackAlpha[300]` as you suggested 😄